src/types/mongoose/cec-data.ts

```diff
@@ -13,6 +13,6 @@ export interface ICECDataMethods {}
 
 interface ICECDataQueryHelpers {}
 
-export interface CECDataModel extends Model<ICECData, ICECDataQueryHelpers, ICECDataMethods> {}
+export type CECDataModel = Model<ICECData, ICECDataQueryHelpers, ICECDataMethods>
```

Comment:
What is the reason for this change? Mostly just curious. This is the model we use everywhere else too

Reply:
Eslint wanted to fix it to be like that 🤷 

I can disable the rule if you like - I don't mind either way

Reply:
I don't have an issue with it, was just curious if there was an actual reason for the change or not. If this makes Eslint happier than sounds good to me. I'll make a note to update the other repos to use this style too for consistency